tests/browser/test/delete_blocks_test.js

```diff
@@ -0,0 +1,226 @@
+const chai = require('chai');
+const {
+  testSetup,
+  testFileLocations,
+  getBlockTypeFromCategory,
+  getSelectedBlockElement,
+  getBlockTypeFromWorkspace,
+  getNumberOfBlocks,
+  getBlockElementById,
+  contextMenuSelect,
+} = require('./test_setup');
+const {Key} = require('webdriverio');
+
+let browser;
+const firstBlockId = '/q5Q_1$kf`gSEFU9C7vK';
+const startBlocks = {
+  'blocks': {
+    'languageVersion': 0,
+    'blocks': [
+      {
+        'type': 'text_print',
+        'id': firstBlockId,
+        'x': 63,
+        'y': 88,
+        'inputs': {
+          'TEXT': {
+            'shadow': {
+              'type': 'text',
+              'id': ']2?G%L#p9Vgca{zd6#JB',
+              'fields': {
+                'TEXT': '1',
+              },
+            },
+          },
+        },
+        'next': {
+          'block': {
+            'type': 'text_print',
+            'id': 'FyV$;[AN7G5frPOrpXg@',
+            'inputs': {
+              'TEXT': {
+                'shadow': {
+                  'type': 'text',
+                  'id': '3q[_v5rA3sl/Vw$*:jGp',
+                  'fields': {
+                    'TEXT': '2',
+                  },
+                },
+                'block': {
+                  'type': 'text_trim',
+                  'id': 'o1O*0%]Vq-4/Nh}2j|wF',
+                  'fields': {
+                    'MODE': 'BOTH',
+                  },
+                  'inputs': {
+                    'TEXT': {
+                      'shadow': {
+                        'type': 'text',
+                        'id': 'A878E^l0J%Hm3iu,j3eV',
+                        'fields': {
+                          'TEXT': 'abc',
+                        },
+                      },
+                      'block': {
+                        'type': 'text',
+                        'id': 'Bma#)g!^Nm_TMHry#ve}',
+                        'fields': {
+                          'TEXT': 'hello',
+                        },
+                      },
+                    },
+                  },
+                },
+              },
+            },
+            'next': {
+              'block': {
+                'type': 'text_print',
+                'id': '{s3Ci:xp;NCsV0(_9ofD',
+                'inputs': {
+                  'TEXT': {
+                    'shadow': {
+                      'type': 'text',
+                      'id': '_zw:|_*p~3itPhy#O*4I',
+                      'fields': {
+                        'TEXT': '3',
+                      },
+                    },
+                  },
+                },
+              },
+            },
+          },
+        },
+      },
+    ],
+  },
+};
+const pauseLength = 20;
+
+suite('Delete blocks', function (done) {
+  // Setting timeout to unlimited as the webdriver takes a longer time to run than most mocha test
+  this.timeout(0);
+
+  // Setup Selenium for all of the tests
+  suiteSetup(async function () {
+    browser = await testSetup(testFileLocations.playground);
```

Comment:
This will break when merged to `develop` due to my refactor in #7291.  Merge/rebase onto `develop` and then fix:

```suggestion
    browser = await testSetup(testFileLocations.PLAYGROUND);
```